packages/console/src/pages/ApplicationDetails/index.tsx

```diff
@@ -1,46 +1,149 @@
+  });
 
   return (
     <div className={styles.container}>
       <BackLink to="/applications">{t('application_details.back_to_applications')}</BackLink>
       {isLoading && <div>loading</div>}
       {error && <div>{`error occurred: ${error.metadata.code}`}</div>}
-      {data && (
-        <Card className={styles.header}>
-          <ImagePlaceholder size={76} borderRadius={16} />
-          <div className={styles.metadata}>
-            <div className={styles.name}>{data.name}</div>
+      {dataFetched && (
+        <>
+          <Card className={styles.header}>
+            <ImagePlaceholder size={76} borderRadius={16} />
+            <div className={styles.metadata}>
+              <div className={styles.name}>{data.name}</div>
+              <div>
+                <div className={styles.type}>{t(`${applicationTypeI18nKey[data.type]}.title`)}</div>
+                <div className={styles.verticalBar} />
+                <div className={styles.text}>App ID</div>
+                <CopyToClipboard value={data.id} className={styles.copy} />
+              </div>
+            </div>
             <div>
-              <div className={styles.type}>{t(`${applicationTypeI18nKey[data.type]}.title`)}</div>
-              <div className={styles.verticalBar} />
-              <div className={styles.text}>App ID</div>
-              <CopyToClipboard value={data.id} className={styles.copy} />
+              <Button title="admin_console.application_details.check_help_guide" />
+            </div>
+          </Card>
+          <Card className={styles.body}>
+            <TabNav>
+              <TabNavLink href={buildTabLink('settings')}>
+                {t('application_details.settings')}
+              </TabNavLink>
+              <TabNavLink href={buildTabLink('advanced-settings')}>
+                {t('application_details.advanced_settings')}
+              </TabNavLink>
+            </TabNav>
+            <div className={styles.tabContent}>
+              <form onSubmit={onSubmit}>
+                <div className={styles.fields}>
+                  {isNormalSettings ? (
+                    <>
+                      <FormField
+                        isRequired
+                        title="admin_console.application_details.application_name"
+                      >
+                        <TextInput {...register('name', { required: true })} />
+                      </FormField>
+                      <FormField title="admin_console.application_details.description">
+                        <TextInput {...register('description')} />
+                      </FormField>
+                      <FormField title="admin_console.application_details.authorization_endpoint">
+                        <CopyToClipboard
+                          className={styles.copy}
+                          value={oidcConfig.authorization_endpoint}
+                        />
+                      </FormField>
+                    </>
+                  ) : (
```

Comment:
Should avoid using long conditional operator expression. Try break it into 2 parts
```typescript
{isNormalSettings && <>...</>}
{!isNormalSettings && <>...</>}
```

Reply:
updated